### PR TITLE
Total element count by Library Type on Homepage

### PR DESCRIPTION
**Pull request by herby2212**
## Description

This feature introduces the option to enable/disable a combined count of elements for each Library Statistics card. _So for each media type._
The feature is disabled by default and can be activated in a new corresponding setting.

Based on issue #1621 .

### Screenshot

**Homepage**
tautulli-total-element-count-homepage

**Settings**
tautulli-total-element-count-setting

## Type of Change

- [x] New feature (non-breaking change which adds functionality)

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have added or updated the docstring for new or existing methods

### CODE REVIEW

File: plexpy/datafactory.py [hunk 1]
```diff
@@ -1031,6 +1031,30 @@ def get_library_stats(self, library_cards=[]):
...
+                total_grandchild_count = 0
+
+                _libraries = library_stats.get(type)
+
+                for library in _libraries:
+                    total_count = total_count + library['count'] if library['count'] != None else total_count
+                    total_child_count = total_child_count + library['child_count'] if library['child_count'] != None else total_child_count
+                    total_grandchild_count = total_grandchild_count + library['grandchild_count'] if library['grandchild_count'] != None else total_grandchild_count
+                
+                #Set the art and thumb to tautulli default -> How to best access it?
```
Comment (herby2212) [inline plexpy/datafactory.py]: What is the best practice to set default art and thumb? For the background in the homepage card when hovering over the "Total" library. _Currently only black for this library._

**Review by herby2212 — COMMENTED** [review]
Fixed in commit https://github.com/Tautulli/Tautulli/pull/1653/commits/1636450483d9b31733c7a19051f6090d9c91a3d6

**Comment by Grim-GH** [comment]
Is this ever going to make it into a release?

**Comment by herby2212** [comment]
> Is this ever going to make it into a release?

This is not up to me, but to @JonnyWong16 .

**Pull request closed without merging**